MeetingNMusae/MeetingNMusae/Views/CharacterBox.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CharacterBox.swift
+//  MeetingNMusae
+//
+//  Created by 김태호 on 2022/06/11.
+//
+
+import Foundation
+import SwiftUI
```

Comment:
```suggestion
import SwiftUI
```

<img width="371" alt="image" src="https://user-images.githubusercontent.com/59143479/173188194-31e148fe-a083-49c7-9e0e-69ea721f22f2.png">

SwiftUI안에 Foundation이 포함되어있어 둘 다 선언할 필요 없습니다!